app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -73,7 +75,13 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                 }
                 is BaseStateViewModel.ViewAction.ShowError -> {
                     binding.refresh.isRefreshing = false
-                    //TODO: handle error here
+
+                    snackbar(requireView(), viewAction.error.message ?: getString(R.string.error_cannot_load_tx_details))
+
+                    if (binding.executed.value.isNullOrBlank() && binding.created.value.isNullOrBlank()) {
```

Comment:
This seems arbitrary to me, could you add a comment why this is done?

Reply:
The reason is to not show the empty screen if we had either executed or created date before. Is there a better way to detect a reload, which had a result before?

Reply:
No I think it is fone, I would just add a code comment to avoid future confusion ;)

Reply:
I will add a comment in the next PR